### ISSUE TITLE
SSH mirror to GitLab Git repository

### ISSUE THREAD

**Issue by falkoneven** [s-kostyaev/stash-mirror-with-key#2]
Good afternoon,

We want to make use of the plugin 'External Hooks' to use the post-receive hook to update another git repository when an commit has been made.
According to github.com/s-kostyaev/stash-mirror-with-key we have tp use the mirror-with-key-debug.sh. For settings we have the following:

**Executable**: mirror-gitlab.sh(which contains the mirror-with-key-debug.sh code), this file is being found.

**Positional Parameters**
ssh://root@falko2.alternet.nl:projecten
/alternet.git
"Private key"

To check if the mirror-gitlab.sh is being called we added the following just below #!/bin/bash:

> echo > start

Which should make a file.. however when we commit nothing seems to happen.. so it seems that the file is not being called after an commit has been made. Does anybody know what could be causing the problem?

**Comment by s-kostyaev** [comment]
Hi,

> however when we commit nothing seems to happen.. so it seems that the file is not being called after an commit has been made. Does anybody know what could be causing the problem?

Try set executeable bit for this file:
`chmod +x mirror-gitlab.sh`
And change 

> echo > start

to smt like this:
`touch /tmp/start`

> Positional Parameters
> ssh://root@falko2.alternet.nl:projecten
> /alternet.git
> "Private key"

Private key must be started on the second line.

**Comment by falkoneven** [comment]
Hi,

Thank you for your help but unfortunately it hasn't been resolved yet, as you suggested I updated my mirror-gitlab.sh to the following:

> #!/bin/bash
> 
> touch /tmp/start

And currently its 0777 so access shouldn't be a problem.
Also in "External Async Post Receive Hook" I cleared everything except Executable. But still it doesn't seem to create an file in the same directory as mirror-gitlab.sh.

Could this /tmp/start folder be made elsewhere or does it still not get called?
